ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -148,7 +148,8 @@
     pool))
 
 (defn- ^Server create-server [options]
-  (let [server (Server. (create-threadpool options))]
+  (let [pool (if (:thread-pool options) (:thread-pool options) (create-threadpool options))
```

Comment:
This could be written more succinctly as:

```clojure
(or (:thread-pool options) (create-threadpool options))
```

Also can you align the `let` clauses?